src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(...args) {
```

Comment:
The `adder` function should only accept a single argument. Using rest parameters (`...args`) here is not necessary and can cause confusion since the function is intended to be called with one argument at a time.